libs/core/dtos/anime-genre.dto.ts

```diff
@@ -0,0 +1,24 @@
+/** Anime genre DTO. */
+export type AnimeGenreDto = {
```

Comment:
- We could reduce the amount of visual noise by marking the whole type as `Readonly` :)

```suggestion
export type AnimeGenreDto = Readonly<{
```